### PR TITLE
Reader: Sites in Search visual updates

### PR DESCRIPTION
**Pull request by jancavan**
### Drop labels as Likes were wrapping when we have 2-cols:

**Before:**
screenshot 2017-06-07 10 01 30

**After:**
screenshot 2017-06-07 11 15 44

### Photo cards with long titles widen the posts column:

**Before:**
screenshot 2017-06-07 11 11 48

**After:**
screenshot 2017-06-07 14 01 04

### Photo cards with long tags widen the posts column:

**Before:**
screenshot 2017-06-07 14 03 37

**After:**
screenshot 2017-06-07 14 10 10

### At ~1012px, only show 3 thumbnails for gallery cards so images don't look squished when we have 2-cols:

**Before:**
screenshot 2017-06-07 12 57 08

**After:**
screenshot 2017-06-07 12 57 27

### CODE REVIEW

**Comment by jancavan** [comment]
### Regular card thumbnails become too narrow at certain breakpoints with 2-cols so stacked them instead:

**Before:**
screenshot 2017-06-07 20 24 43

**After:**
screenshot 2017-06-07 20 24 51

### Moved Settings icon lower so tap targets aren't too close:

**Before:**
screenshot 2017-06-07 20 26 46

**After:**
screenshot 2017-06-07 20 26 53

### Fix alignment of Follow(ing) buttons in single-col site results:

**Before:**
screenshot 2017-06-07 20 41 03

**After:**
screenshot 2017-06-07 20 41 27

File: client/reader/_style.scss [hunk 1]
```diff
@@ -23,7 +23,6 @@
 .is-reader-page .follow-button {
 	border: 0;
 	border-radius: 0;
-	float: right;
 }
```
Comment (jancavan) [inline client/reader/_style.scss]: - [ ] Make sure Follow(ing) buttons in other pages are still floated right - ie Tags page.

**Comment by jancavan** [comment]
- [ ] Fix post recs top margin
- [ ] Make sure Follow(ing) buttons in other pages are still floated right - ie Tags page
- [ ] Cleanup

**Comment by fraying** [comment]
Lookin good, @jancavan!

**Comment by jancavan** [comment]
https://github.com/Automattic/wp-calypso/issues/14878 is fixed in https://github.com/Automattic/wp-calypso/pull/14881/commits/5a111ffa8f59a231204d59cb465d35a70a32bebc:

**Before:**
screenshot 2017-06-08 20 41 42

**After:**
screenshot 2017-06-08 20 41 46

The URL is go longer before we add the fade. I'll fix that in a separate PR.

**Comment by jancavan** [comment]
https://github.com/Automattic/wp-calypso/issues/14875 is also fixed in https://github.com/Automattic/wp-calypso/pull/14881/commits/5a111ffa8f59a231204d59cb465d35a70a32bebc:

**Before:**
screenshot 2017-06-08 20 46 19

**After:**
screenshot 2017-06-08 20 46 13

File: client/reader/_style.scss (hunk 1 above)
Comment (jancavan) [inline client/reader/_style.scss]: Done in: https://github.com/Automattic/wp-calypso/pull/14881/files#diff-418e0c0a76dea69ba95b1efe94eaf91eR11

**Comment by fraying** [comment]
Nice work, @jancavan! Note that this is still to do:

Lines under headers are 2px and lighter grey. Should be 1px and darker grey.

**Comment by samouri** [comment]
@jancavan: I'm not sure how to rebase this. I'm leaning towards just keeping this version outright and making modifications to that as we need to

**Comment by blowery** [comment]
> I'm leaning towards just keeping this version outright and making modifications to that as we need to

mostly looks like concurrent updates. I don't think keeping just this one would work...

**Comment by samouri** [comment]
> mostly looks like concurrent updates. I don't think keeping just this one would work...

Yep. just checked, it'll require tweaking.

**Comment by fraying** [comment]
Still need the top lines darker.

Current:

before


Should be:

after

**Comment by fraying** [comment]
Should be less space between search box and headers, more space between headers and lines, and less space between lines and first result. All three whitespace areas should be the same (~20px).

Current:

before


Should be:

after

**Comment by fraying** [comment]
At middle widths, on Sites, the Follow buttons are misaligned.

screen shot 2017-06-13 at 2 48 24 pm

**Comment by fraying** [comment]
My two cents: We shouldn't have lines in between the site results. They're not really needed and they make the page feel overwhelmed with lines.

**Comment by fraying** [comment]
Spacing is broken at the smallest width.

screen shot 2017-06-13 at 3 05 28 pm

**Comment by jancavan** [comment]
Superseded by: https://github.com/Automattic/wp-calypso/pull/14881

**Pull request closed without merging**